src/styles/uiConfigurations.css

```diff
@@ -71,7 +71,7 @@
 
 .logo {
   position: fixed;
-  height: 75px;
-  left: 0%;
+  height: 60px;
```

Comment:
Since you will be reverting height of .toolbar to 'max-content', the changes on line 74 and 75 can be reverted as well.